app/components/shared/team-members.tsx

```diff
@@ -0,0 +1,43 @@
+import { type User } from "@prisma/client"
+
+import { Card } from "~/components/ui/card"
+
+export function TeamMembers({ teamMembers }: { teamMembers: User[] }) {
+	if (teamMembers.length <= 0) {
+		return <p>There are no members yet.</p>
+	}
+
+	return (
+		<div className="space-y-4">
+			<ul className="grid grid-cols-1 gap-4 md:grid-cols-2">
+				{teamMembers.map(member => (
+					<li key={member.id}>
+						<Card className="flex h-full flex-col gap-4 p-4">
+							<img
+								className="h-32 w-32 rounded-full object-cover"
```

Comment:
@fikrialwan avatar image can be a bit smaller to be "h-20 w-20" or need some sense so it's just right, not too big, not too small.

Reply:
I adjust avatar image to be "h-24 w-24", I think that size is not too big and not too small